apps/codeforafrica/src/pages/api/v1/draft/disable-draft.js

```diff
@@ -0,0 +1,5 @@
+// By default, the Draft Mode session ends when the browser is closed.
+// This method clears it manually / on demand.
```

Comment:
on what instances might we need to use this function @kilemensi ?

Reply:
https://nextjs.org/docs/pages/building-your-application/configuring/draft-mode#clear-the-draft-mode-cookie @koechkevin